contracts/validator-manager/interfaces/IPoSValidatorManager.sol

```diff
@@ -131,6 +129,14 @@ interface IPoSValidatorManager is IValidatorManager {
      */
     function completeDelegatorRegistration(uint32 messageIndex, bytes32 delegationID) external;
 
+    /**
+     * @notice Removes a delegator from a completed validation period. The delegator can be in either the active or pending removed
+     * state. No uptime proof is required in this case, because it will have been provided by the validator upon their exit.
+     * Note that this function can only be called by any address to clean up the delegation.
```

Comment:
"can only be called by any address" seems a bit contradictory 😅 

Think there is arguments to be made that either it should only be able to be called by the owner of the delegation, or that it should be callable by anyone, but lets clarify either way.